tests/accumulator_test.py

```diff
@@ -293,23 +293,24 @@ def compute_metrics(self):
 class CountAccumulatorTest(unittest.TestCase):
 
     def test_without_noise(self):
+        no_noise = (1, 1, 0, 0, 0, 0, pipeline_dp.NoiseKind.GAUSSIAN)
```

Comment:
Please add a test with noise. It's enough to have one test case. 
Here is an [example](https://github.com/OpenMined/PipelineDP/blob/1505b288b620b019080eca5c7b90915633bc45cd/tests/accumulator_test.py#L342) how to have test with noise

Reply:
Done.